docker-compose.yml

```
@@ -0,0 +1,26 @@
+version: "3.8"
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)